config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -76,6 +76,46 @@ periodics:
       securityContext:
         privileged: true
 
+- interval: 6h
+  name: ci-kubernetes-e2e-aks-engine-azure-1-14-windows
+  labels:
+    preset-service-account: "true"
+    preset-azure-acsengine: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190301-76bc03340-master
+      args:
+      - "--job=$(JOB_NAME)"
+      - "--root=/go/src"
+      - "--repo=k8s.io/kubernetes=release-1.14"
```

Comment:
Question for reviewers: will this branch receive updates daily?

Reply:
Yes, @hoegaarden is responsible for this and post in #sig-release when they're done